### DISCUSSION TITLE
Integrate with automation services

### DISCUSSION THREAD

**Discussion by amblerkr** [dubinc/dub#148] [Ideas]
This is the best.
[Suggest some features]

1. Integrate with automation services (IFTTT, Zapier, Make, MS Automate, Open Source: Huginn (https://github.com/huginn/huginn))

2. Export All (Google Sheets or Airtable)
https://news.ycombinator.com/item?id=32941168

3. Can I change my avatar to Gravatar? Or can I add an avatar that I want? If image hosting is a problem, it would be good to bring an image of Github Or make use of the image URL.

**Comment by fmerian** [comment]
great suggestions, @amblerkr!

some additional thoughts:

> 1. Integrate with automation services (IFTTT, Zapier, Make, MS Automate, Huginn)

what would be your main use cases (triggers and actions)? I'd add n8n -- open-source alternative to Zapier -- to the list. This might be solved with Issue #73.

> 2. Export All (Google Sheets or Airtable)

like in the [HN thread](https://news.ycombinator.com/item?id=32941168) you mentioned, is the objective to export data in case Dub shuts down?

> 3. Can I change my avatar to Gravatar? Or can I add an avatar that I want?

like the idea, too. Do you think it's a must-have for the product?

**Reply by amblerkr** [reply]
Thanks.

1. I know n8n. here!
ⓐ RSS → dub → Webhook(Discord, Slack)
ⓑ Discord Server Channel #dub → Discord Channel #updates
Automatically post to channels that notify you of the news
ⓒ RSS → Filter for specific keywords  → dub → Webhook(Discord, Slack)

2. Yes. I can also back up in case the service ends. In addition, due to the nature of the job, it is often necessary to export and provide measurements to others.

3. It is not necessary, but it will be effective if it is made up of teams.
I can add a team-specific profile background color to your account profile or put a specific marker. This can be resolved by adding more detailed team functions.

**Reply by fmerian** [reply]
thanks, amber :)

> - RSS → dub → Webhook(Discord, Slack)
> - Discord Channel #dub → Discord Channel #updates - Automatically post to channels that notify you of the news
> - RSS → Filter for specific keywords  → dub → Webhook(Discord, Slack)

so, to clarify - sorry for that! - you'd like:

1. When a new RSS feed item is published, then
2. Create a Dub link, then
3. Post it (on Discord, Slack...).

is it correct?

> In addition, due to the nature of the job, it is often necessary to export and provide measurements to others.

what do you think of embeds? using a service like [embed.ly](https://embed.ly/) would ease the export/import flow.

**Reply by amblerkr** [reply]
You're right. Also, when I post on a particular discord channel (staff only), post it on a public channel.

> what do you think of embeds? using a service like [embed.ly](https://embed.ly/) would ease the export/import flow.

That's a good function, too. It is suitable for temporary or small projects.
I prefer to have the data. Exporting is a higher priority.
If it is exported, it will be easy for me to filter it and edit it in a good form. Excel OR Google Sheets

**Reply by amblerkr** [reply]
If a new shorten URL is created, save it in Google Sheet.

**Comment by steven-tey** [comment]
Oh I love this! Planning to release the Dub API soon so we'll be able to start building out some awesome integrations soon! First one I have in mind is Raycast – I use it daily so it'll be super convenient to just paste in a link and get a short URL back! Will keep y'all posted! 🤩

**Reply by fmerian** [reply]
> First one I have in mind is Raycast – I use it daily so it'll be super convenient to just paste in a link and get a short URL back!

LOVE IT! A Raycast extension would be amazing indeed, with multiple use cases:
- Shorten URL from clipboard
- Quick access to latest shorten links
- Get analytics from latest shorten links

🔥
